include/rcpputils/filesystem_helper.hpp

```diff
@@ -137,6 +137,17 @@ class path
     return split_fname.size() == 1 ? path("") : path("." + split_fname.back());
   }
 
+  path remove_extension(int n_times = 1)
```

Comment:
Add documentation on what `n_times` means and the significance of a default value of `1`

Reply:
Also add documentation on what the returned value is and if its a copy of the object